css/tideline.less

```diff
@@ -430,12 +422,29 @@ svg {
       stroke: @basal-automated;
     }
   }
+
   // only in legends
-  &.d3-rect-basal-undelivered {
-    fill: none;
-    stroke: @basal;
-    stroke-dasharray: 1.5, 3;
-    stroke-width: 1.5;
+  &.d3-basal-loop-mode {
```

Comment:
replacing d3-basal-automated with d3-basal-loop-mode is breaking the display of the automated basal
![image](https://user-images.githubusercontent.com/22294614/76086239-01f7f200-5fb4-11ea-8ccd-fd6337d3d261.png)

Reply:
Technically I didn't replace d3-basal-automated with d3-basal-loop-mode.
I removed "d3-basal-automated" and add "d3-basal-loop-mode"
I don't see what is broken. Can you provide a screenshot with difference (same as above but without the problem) ?